### ISSUE TITLE
Can I see the training LOG (POse Estimation)??

### ISSUE THREAD

**Issue by Dipankar1997161** [xingyizhou/CenterNet#980]
@xingyizhou 
Hey, Could I possibly see the training log for human pose estimation?? 

It would be helpful to understand if I am not overfitting. My training and Val loss are both converging well but wanted to check the final loss value.
I am using both DLA1x and HG 1x. 

Hg_1x converged faster as compared to DLA1x, is this normal or am I misinterpreting something. Hence I request the training log.
The screenshot is for DLA1x, I am using my custom dataset, and it still converges without overfitting.

![Screenshot 2022-05-06 110220](https://user-images.githubusercontent.com/85514219/167101448-68e46e89-29a7-4a3a-9fdb-7939a36fcd1b.png)
Thanks in advance.
